tutoroars/plugin.py

```diff
@@ -33,6 +33,8 @@
         ("OARS_EVENT_SINK_RELATIONSHIPS_TABLE", "course_relationships"),
         ("OARS_EVENT_SINK_OVERVIEWS_TABLE", "course_overviews"),
         ("OARS_EVENT_SINK_CLICKHOUSE_TIMEOUT_SECS", "5"),
+        # ClickHouse tracking log settings
+        ("OARS_TRACKING_LOG_DATABASE", "openedx"),
```

Comment:
This may not be the name we want to settle on given the pattern of the xapi database being separate, but I wanted to change as little as possible in initial testing.